python/materials.py

```diff
@@ -258,12 +258,17 @@
 Ag_frq4 = 9.083*eV_um_scale      # 0.137 um
 Ag_gam4 = 0.916*eV_um_scale
 Ag_sig4 = Ag_f4*Ag_plasma_frq**2/Ag_frq4**2
+Ag_f5 = 5.646
+Ag_frq5 = 20.29*eV_um_scale      
```

Comment:
This additional Lorentzian term corresponds to a resonance at a wavelength of 0.061 μm (or 61 nm) that is so far outside the optical range that it should not affect results but it will increase the computational expense due to the additional storage and time stepping of auxiliary fields. (This is why it was originally left out.) Is it really necessary?

Reply:
Right, but it seems to be a pretty broad resonance because even though it's so far away, it does alter the shape of the resonances between 100 nm and 300 nm. After that, however, it's negligible.